content/user/first_steps/basic_terms.md

```diff
@@ -64,13 +64,19 @@ is just
 
 `/includes/languages/english/`
 
+### YOURACCOUNT
+The variable `YOURACCOUNT` is used to refer to the name yoru hosting company
```

Comment:
Typo: `to the name yoru hosting company`
- "yoru"
- missing "of"

Reply:
Thanks will fix next checkin